linera-core/src/updater.rs

```diff
@@ -31,9 +31,9 @@ use crate::{
     remote_node::RemoteNode,
 };
 
-/// The amount of time we wait for additional validators to contribute to the result, as a fraction
-/// of how long it took to reach a quorum.
-const GRACE_PERIOD: f64 = 0.2;
+/// The default amount of time we wait for additional validators to contribute 
+/// to the result, as a fraction of how long it took to reach a quorum.
+const DEFAULT_GRACE_PERIOD: f64 = 0.1;
```

Comment:
Please keep the current value

Reply:
The value was changed following the advice in #2836.

Reply:
I don't have a strong opinion; sorry for the confusion!
(The idea was that since `0.2` means we make the _happy path_ 20% slower it improves user experience to reduce this. And we will need additional tools to help slow validators catch up anyway. But we can always change the default later.)